www/gatsby-node.js

```diff
@@ -358,17 +352,40 @@ exports.createPages = ({ graphql, actions, reporter }) => {
         if (!slug) return
 
         if (!_.includes(slug, `/blog/`)) {
-          createPage({
-            path: localizedPath(locale, node.fields.slug),
-            component: slash(
-              node.fields.package ? localPackageTemplate : docsTemplate
-            ),
-            context: {
-              slug: node.fields.slug,
-              locale,
-              ...getPrevAndNext(node.fields.slug),
-            },
-          })
+          const prevAndNext = getPrevAndNext(node.fields.slug)
+
+          if (node.frontmatter.jsdoc) {
+            // API template
+            createPage({
+              path: localizedPath(locale, node.fields.slug),
+              component: slash(apiTemplate),
+              context: {
+                slug: node.fields.slug,
+                jsdoc: node.frontmatter.jsdoc,
+                apiCalls: node.frontmatter.apiCalls,
+                ...prevAndNext,
+              },
+            })
+          } else if (node.fields.package) {
+            // Local package template
+            createPage({
+              path: `${node.fields.slug}`,
+              component: slash(localPackageTemplate),
+              context: {
+                slug: node.fields.slug,
+              },
+            })
+          } else {
+            // Docs template
+            createPage({
+              path: localizedPath(locale, node.fields.slug),
+              component: slash(docsTemplate),
+              context: {
+                slug: node.fields.slug,
```

Comment:
```suggestion
                slug: node.fields.slug,
                locale,
```

(missing locale here cause thousands of query to fail because it is required variable in those queries)